2024/opmodes/red/CenterstageRedRightParking.java

```diff
@@ -0,0 +1,259 @@
+        goRightToObject.stop();
+        goLeftToObject.stop();
+
+
+        outtakePath = new DeadReckonPath();
+        outtakePath.addSegment(DeadReckonPath.SegmentType.STRAIGHT, OUTTAKE_DISTANCE, OUTTAKE_SPEED);
+
+        //addSegment adds a new segment or direction the robot moves into
+
+        //robot moves to the right into the backstage parking area
+        goToPark.addSegment(DeadReckonPath.SegmentType.SIDEWAYS, RIGHT_DISTANCE, DRIVE_SPEED);
+
+        //robot moves to the object in the right
+        goRightToObject.addSegment(DeadReckonPath.SegmentType.STRAIGHT, 4, DRIVE_SPEED);
+        goRightToObject.addSegment(DeadReckonPath.SegmentType.TURN, 4, DRIVE_SPEED);
+
+        //robot moves to the object in the front
+        goStraightToObject.addSegment(DeadReckonPath.SegmentType.STRAIGHT, 4, DRIVE_SPEED);
+
+        //robot moves to the object in the left
+        goLeftToObject.addSegment(DeadReckonPath.SegmentType.STRAIGHT, 4, DRIVE_SPEED);
+        goLeftToObject.addSegment(DeadReckonPath.SegmentType.TURN, 4, -DRIVE_SPEED);
+
+    }
+
+    //initializes the declared motors and servos
+    @Override
+    public void init()
+    {
+        //initializes the motors for the wheels
+        frontLeft = hardwareMap.get(DcMotor.class, "frontLeft");
+        frontRight = hardwareMap.get(DcMotor.class, "frontRight");
+        backLeft = hardwareMap.get(DcMotor.class, "backLeft");
+        backRight = hardwareMap.get(DcMotor.class, "backRight");
+
+        //sets wheel motors to run using the encoders
+        frontLeft.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        frontRight.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        backLeft.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        backRight.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+
+        //initializes drivetrain, clears the encoder values, and prepares motors to run on the encoders
+        drivetrain = new FourWheelDirectDrivetrain(frontRight, backRight, frontLeft, backLeft);
+        drivetrain.resetEncoders();
+        drivetrain.encodersOn();
+
+        //displays telemetry of robot location
+        whereAmI = telemetry.addData("location in code", "init");
+
+        //initializes motor mechanism, returns what motor would do if 0 power behavior was implemented on it,
+        //rests encoder, and prepares motors to run on the encoders
+        outtake = hardwareMap.get(DcMotor.class, "outtake");
+        outtake.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
+        outtake.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        outtake.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+
+        //initializes the motor drivetrain, resets encoders, and prepares motor(s) to run on the encoders
+        outtakeDrivetrain = new OneWheelDirectDrivetrain(outtake);
+        outtakeDrivetrain.resetEncoders();
+        outtakeDrivetrain.encodersOn();
+
+        //calls method to start the initialization
+        initPaths();
+
+
+    }
+
+
```

Comment:
I think this file is missing the OpenCVPipeline calls. Do you want to add it before merging the pull request?